robyn/types.py

```diff
@@ -18,5 +20,34 @@ def as_list(self):
         ]
 
 
+PathParams = NewType("PathParams", Dict[str, str])
+Method = NewType("Method", str)
+URL = NewType("URL", RustUrl)
+FormData = NewType("FormData", Dict[str, str])
+Files = NewType("Files", Dict[str, bytes])
+IPAddress = NewType("IPAddress", Optional[str])
+Identity = NewType("Identity", Optional[RustIdentity])
```

Comment:
can we not use the builtin types for these?

Reply:
only url & identity? yep. but, we should have type aliases for other types.

Reply:
@sansyrox PTAL now.